repose-aggregator/services/datastore/impl/src/main/java/org/openrepose/core/services/datastore/impl/ehcache/EHCacheDatastore.java

```diff
@@ -66,6 +66,7 @@ public void put(String key, Serializable value) {
 
     @Override
     public void put(String key, Serializable value, int ttl, TimeUnit timeUnit) {
+        // todo: does not allow for eternal caching (need to either setEternal(true) or set ttl and tti to 0
```

Comment:
That may be true in this filter, although I would never presume to know what craziness people are using my code for. Even so, this datastore is used in other parts of our code and is available as a service to any filter developer. It's a pretty easy fix, so it shouldn't take long, and it would make our EHCacheDatastore facade mirror the behavior of an Ehcache instance more closely.

Granted, anytime you want to cache something in memory indefinitely, Java variables work just as well (unless you need to do something crazy).

Ultimately, I think this is a quick win, but low priority.